### ISSUE TITLE
[Improvement] Ability to use a proxy with the updater

### ISSUE THREAD

**Issue by natinusala** [libretro/RetroArch#2101]
Hello,

I think it would be interesting to be able to fill in proxy settings into the config file in order to use the updater where a web proxy is needed, because now it just hang and eventually fail the download.

**Comment by matcarfer** [comment]
Absolutely. RetroArch is useless for me as I'm behind a proxy and RetroArch doesn't come with cores anymore. It should have all cores available (except the ones that have licensing issues) and proxy support.

**Comment by xTMODx** [comment]
also with tor/orbot on android it needs proxy settings too

**Comment by ademircerqueira1** [comment]
Almost ten years and nothing!

**Comment by matcarfer** [comment]
> Almost ten years and nothing!

this needs someone, in his/her free time, to work, for us, for free. so maybe, you could do it or ask politely. i love retroarch and love everyone involved, so this attitude doesnt help, at all. I also want this, but its no good to demand it.

**Comment by ademircerqueira1** [comment]
I can work to help! Where do I start?

> > Almost ten years and nothing!
> 
> this needs someone, in his/her free time, to work, for us, for free. so maybe, you could do it or ask politely. i love retroarch and love everyone involved, so this attitude doesnt help, at all. I also want this, but its no good to demand it.

**Comment by lucatrv** [comment]
To run RetroArch behind a web proxy, you can use [ProxyChains](https://github.com/rofl0r/proxychains-ng).
